dashboard/app/models/concept.rb

```diff
@@ -56,11 +53,16 @@ def self.setup
   def self.setup_with_concepts(concepts_by_index)
     videos_by_concept = Video.where(key: concepts_by_index).index_by(&:key)
     concepts = concepts_by_index.map.with_index(1) do |concept, id|
-      {id: id, name: concept, video_id: videos_by_concept[concept]&.id}
+      {id: id, name: concept, video_key: videos_by_concept[concept]&.key}
```

Comment:
Why dd you decide to switch from video_id to video_key for concept videos?

Reply:
The logic as it is before assumes there's a 1:1 mapping from concept to an individual video, which isn't true with dubbed videos. However, there is still a 1:1 mapping from concept to video key as dubbed videos have the same key as their English counterpart. This changes allows us to show the video associated with a concept based on the language of the user.